config.js

```diff
@@ -3,11 +3,8 @@ export const API_URL =
     ? 'https://api.core.ac.uk/internal'
     : 'https://api.dev.core.ac.uk/internal'
 
-const ORIGIN = encodeURIComponent(
-  typeof window == 'object' ? window.location.origin : ''
-)
-
-export const LOGIN_URL =
-  process.env.NODE_ENV === 'production'
-    ? '/login.html'
-    : new URL(`/test/login?continue=${ORIGIN}`, API_URL).toString()
+export const getLoginPage = (path = window.location.origin) =>
```

Comment:
If this is called on server, `window` is undefined. I would keep the following.

```suggestion
export const getLoginPage = (path = window?.location.origin) =>
```

Reply:
this won't help since window is not defined. Let's keep it as is till we run into this issue...